src/main/java/com/hackclub/hccore/commands/StatsCommand.java

```diff
@@ -195,6 +195,5 @@ private static String toSIPrefix(double number) {
       number /= 100;
       return String.format("%.2f k", number);
     }
-    return null;
```

Comment:
Would prefer this is converted into an if-elif-else, just so on reading it's even more clear that this can never occur, and the reader doesn't have to worry about off by one errors etc to see if it would implicitly return null